_posts/2018-11-02-project-minimalistic-documentation.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: Project documentation 
+description: "Minimalistic but useful project documentation."
+author: bogdan
+modified: 2014-12-24
+tags: [documentation, agile, process, qa]
+categories: [Process]
+image:
+    feature: posts/project_documentation/title.jpg
+---
+
+We don’t like to write documentation. But we do like good documentation for tools we use. Which is a little bit controversial, isn’t it? Actually, this mean that we are too lazy to write docs, but we understand that it’s necessary for any good project. Here is few tips  how to create a “minimalistic but useful” documentation  from guy that don’t like to write documentation too. Hope it will be helpful.
```

Comment:
We don’t like writing documentation. What we do like is having good documentation for tools we use. Which is a little bit controversial, isn’t it? Actually, this means that most likely we are too lazy to produce docs, at the same time we understand that it is necessary for any good project. Here are a few tips on how to create a “minimalistic but useful” documentation  from a guy who doesn’t like to write it; I hope it will be helpful.

Reply:
I put in italic an alternative version of saying the same thing but in a cheeky manner. Please take it off it this is not what you meant to say :-)